.gitignore

```diff
@@ -15,15 +15,15 @@ examples/CocoapodsDemo/ios/CocoapodsDemo.xcodeproj/xcuserdata
 examples/CocoapodsDemo/ios/CocoapodsDemo.xcworkspace/xcshareddata
 examples/CocoapodsDemo/ios/CocoapodsDemo.xcworkspace/xcuserdata
 examples/CocoapodsDemo/ios/Pods/Pods.xcodeproj/xcuserdata
-examples/RNOneSignal/android/app/build
-examples/RNOneSignal/android/.gradle
-examples/RNOneSignal/android/.idea
-examples/RNOneSignal/android/build
-examples/RNOneSignal/ios/RNOneSignal.xcodeproj/project.xcworkspace/xcshareddata
-examples/RNOneSignal/ios/RNOneSignal.xcodeproj/project.xcworkspace/xcuserdata
-examples/RNOneSignal/ios/RNOneSignal.xcodeproj/xcuserdata
-examples/RNOneSignal/ios/RNOneSignal.xcworkspace/xcshareddata
-examples/RNOneSignal/ios/RNOneSignal.xcworkspace/xcuserdata
+examples/RNOneSignalTS/android/app/build
```

Comment:
do we intend to replace the other example project?

Reply:
yes, after discussing with @jkasten2 we decided to remove the other one